resources/js/components/Cart/Cart.vue

```diff
@@ -37,6 +37,15 @@
                     Notify(error.response.data.message, 'error')
                 }
             }
+        },
+        created() {
+            this.$root.$once('refresh-cart', () => {
+                this.refreshCart()
+            });
+
+            this.$root.$once('clear-cart', () => {
```

Comment:
`clearCart(keys = [])` also supports an array of keys, maybe it's useful to add it as a parameter to the event listener as well